dapp/src/components/buySell/ApproveSwap.js

```diff
@@ -63,13 +56,36 @@ const ApproveSwap = ({
     ousd,
   } = useStoreState(ContractStore, (s) => s.contracts || {})
 
-  const contracts = {
-    vault: vault,
-    flipper: flipper,
-    uniswap: uniV3SwapRouter,
-    curve: curveOUSDMetaPool,
-    uniswapV2: uniV2Router,
-    sushiswap: sushiRouter,
+  function alterFirstLetterCase(string, forceUpperCase) {
+    return forceUpperCase ? string.charAt(0).toUpperCase() : string.charAt(0) + string.slice(1);
+  }
+
+
+  const routeConfig = {
```

Comment:
added also names to the `contracts` mappings for easier understanding

Reply:
Also name is a function with a boolean where we can easily force it to become uppercase